zerocopy-derive/src/lib.rs

```diff
@@ -107,6 +118,63 @@ const STRUCT_UNION_ALLOWED_REPR_COMBINATIONS: &[&[StructRepr]] = &[
     &[StructRepr::C, StructRepr::Packed],
 ];
 
+// A struct is `FromZeroes` if:
+// - all fields are `FromZeroes`
+
+fn derive_from_zeroes_struct(ast: &DeriveInput, strct: &DataStruct) -> proc_macro2::TokenStream {
+    impl_block(ast, strct, "FromZeroes", true, PaddingCheck::None)
+}
+
+// An enum is `FromZeroes` if:
+// - Every possible bit pattern must be valid, which means that every bit
```

Comment:
Does this comment need fixup? Or is this just a stricter guarantee for now until we go back and fix it up?

Reply:
The latter - right now, this is just a copy-paste from the comment on the `FromBytes` derive.